tests/test_digitaltwin/test_setup_environment.py

```diff
@@ -1,28 +1,38 @@
 import os
+import platform
 import unittest
 
+import pytest
 from sqlalchemy.exc import OperationalError
 
 from src.digitaltwin import setup_environment
 
 
 class SetupEnvironmentTest(unittest.TestCase):
 
-    @unittest.skip("Skipping until we work on https://github.com/GeospatialResearch/Digital-Twins/issues/23")
+    @pytest.mark.skipif(
+        platform.system() != "Windows",
```

Comment:
What if the developer uses other operating systems, such as Linux, for local development? Same goes for the one below.

Reply:
Casey and I are having a discussion about implementing a environment variable to be able to switch off these types of tests that require the database, so that they can be avoided only on the test runner or other special cases as needed. I think this could be a good idea, but am wary of having developers turn it off for convenience and never turning it back on. I will implement it for review